query-engine/connectors/sql-query-connector/src/database/operations/write.rs

```diff
@@ -1,11 +1,15 @@
 use crate::{error::SqlError, query_builder::write, QueryExt};
 use connector_interface::*;
+use itertools::Itertools;
 use prisma_models::*;
 use prisma_value::PrismaValue;
 use quaint::error::ErrorKind;
-use std::{collections::HashMap, convert::TryFrom};
+use std::{collections::HashMap, convert::TryFrom, usize};
 use user_facing_errors::query_engine::DatabaseConstraint;
 
+/// The maximum amount of rows that can be inserted at once.
+const MAX_ROWS: usize = 2500;
```

Comment:
Hey dom, out of curiosity where does this number come from?

Reply:
SQL  server has a limit of rows, as per @pimeys.

Reply:
Yeah that's something I read from a Tiberius ticket. Haven't tested it, but inserting 2500 records with a normal query is going to be quite catastrophic anyhow...

Reply:
Thanks